packages/docs/celo-codebase/protocol/governance.md

```diff
@@ -29,7 +29,12 @@ Every day the top three proposals at the head of the queue are popped off and mo
 
 ## Referendum
 
-Once the Approval phase is over, approved proposals graduate to the referendum phase. Any user may vote yes, no, or abstain on these proposals. Their vote's weight is determined by the weight of their Locked Gold commitment. After the Referendum phase is over, which lasts two days, each proposal is marked as passed or failed as a function of the votes and the corresponding passing function parameters. These parameters specify what is required for specific proposals to pass. In the Alfajores testnet, this is a simple majority, but in future the voting threshold will adapt based on participation levels.
+Once the Approval phase is over, approved proposals graduate to the referendum phase. Any user may vote yes, no, or abstain on these proposals. Their vote's weight is determined by the weight of their Locked Gold commitment. After the Referendum phase is over, which lasts two days, each proposal is marked as passed or failed as a function of the votes and the corresponding passing function parameters.
```

Comment:
Do the docs use "Locked Gold commitment" anywhere else?

Reply:
I am not sure. I did not add that language, but I can certainly change it if we want.